src/SprykerEco/Yves/Unzer/Theme/default/components/molecules/unzer-credit-card/unzer-credit-card.scss

```diff
@@ -0,0 +1,22 @@
+@mixin unzer-unzer-credit-card($name: '.unzer-credit-card') {
+    #{$name} {
+        .unzerUI {
```

Comment:
Are all these classes from an external library?

Reply:
Yes they are, I'm not able to manage them.